src/components/CookieBanner/utils/getCookie.js

```diff
@@ -0,0 +1,6 @@
+const getCookie = (cname) => {
+  const match = document.cookie.match(`(^|;) ?${cname}=([^;]*)(;|$)`)
+  return match ? match[2] : ''
```

Comment:
Maybe we should use a small library to interact with cookies? Worried this could break in certain edge cases

Reply:
https://www.npmjs.com/package/js-cookie